### ISSUE TITLE
List of Commonalities

### ISSUE THREAD

**Issue by fedorov** [danielskatz/software-vs-data#46]
I came across this effort while reading https://peerj.com/articles/cs-86/, and my first thought after going over the list - where to start? So many differences, that I am asking myself how could it be possible for someone to be in any doubt about the differences between software and data?

For someone coming this this perspective, and to make the document more balanced and motivated, did you consider adding "List of Commonalities"?

**Comment by danielskatz** [comment]
Hi - this really started when a draft of that paper discussed differences between software and data from the point-of-view of citation, and we want to explain when the data citation principles were not sufficient and correct for software citation. Some reviewers felt we were injecting out opinions, rather than facts, so we decided to create this repo and let people discuss this, so it was more of a consensus and not just out opinions.  We then could cite this repo in the paper, and satisfy the reviewers, which we did.

Having said that, if you want to propose some changes, that would be fine.
